elements/rh-navigation-secondary/rh-navigation-secondary.css

```diff
@@ -58,24 +53,19 @@ nav.rtl {
   grid-template-columns: 1fr max-content;
   grid-template-areas:
     "logo menu"
-    "nav nav"
-    "cta cta";
+    "provider provider";
```

Comment:
on call we talked about renaming this

options:

- main
- nav
- menu
- mobile menu

anything other than provider, really

Reply:
fixed in [01d6710](https://github.com/RedHat-UX/red-hat-design-system/pull/1108/commits/01d67103b133562ff0c9ddcb1c074f0de08d80c1)